### PR TITLE
FIXED DA BOOG

### PR DESCRIPTION
**Pull request by kelvindo**
fix fix fix da boog boog booger

### CODE REVIEW

File: lambda_function.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def parse_highlights(highlights):
         quote_indices = []
         for x in range(num_quotes):
             # no repeats
-            quote_index = 0
-            while quote_index in quote_indices:
```
Comment (kelvindo) [inline lambda_function.py:34]: 0 is not in quote_indicies so we always add 0 before selecting a random number

Comment (kelvindo) [reply lambda_function.py:34]: dont hide this

Comment (kelvindo) [reply lambda_function.py:34]: its a real boog

**Review by japalio — APPROVED** [review]

**Comment by japalio** [comment]
Good catch!

**Comment by kelvindo** [comment]
DEPLOY NAO @japalio

**Pull request merged**